src/Aquifer.Jobs/SendNewContentEmails.cs

```diff
@@ -20,7 +20,9 @@ public class SendNewContentEmails(
     TelemetryClient telemetryClient)
 {
     [Function(nameof(SendNewContentEmails))]
-    public async Task Run([TimerTrigger("%MarketingEmail:CronSchedule:NewContent%")] TimerInfo _, CancellationToken ct)
+#pragma warning disable IDE0060 // Remove unused parameter: A (non-discard) TimerInfo parameter is required for correct Azure bindings
```

Comment:
I don't like doing this manually for each such parameter but I still think it's preferable to leave IDE0060 enabled.